Makefile

```diff
@@ -134,7 +135,11 @@ test-coverage: ${GOLANGCOVER} ## Run tests with coverage
 	go test -run . $(PKG) -coverprofile=coverage.txt -covermode=atomic
 
 ${GOLANGCILINT}:
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin v1.45.2
+ifeq ($(HAS_LINT),false)
```

Comment:
Why are we removing this?

Reply:
because it fails

Reply:
also they have different instructions for installing locally vs in a CI/CD pipeline.